crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -577,6 +578,28 @@ impl SweetConductor {
         Ok(clone)
     }
 
+    /// Call into the underlying update_coordinators function and update the
+    /// associated DnaFile in the SweetConductor such that the up-to-date DnaFile
+    /// will be loaded into the RibosomeStore in case of a conductor restart.
```

Comment:
Add links to doc-comments (not sure if these are all correct and it might need reformatting now)

```suggestion
    /// Call into the underlying [`DnaFile::update_coordinators`] function and update the
    /// associated [`DnaFile`] in the [`SweetConductor`] such that the up-to-date [`DnaFile`]
    /// will be loaded into the [`RibosomeStore`] in case of a conductor restart.
```

Reply:
Thanks, addressed: https://github.com/holochain/holochain/compare/6366cf77e2f65ac144663f3f9a36a2d5ad10acf1..8eeb460d522713ece99979499294d27c8f207245